### PR TITLE
Update for CUDA12.5.0

### PR DESCRIPTION
**Pull request by adibbley**
xref: https://github.com/conda-forge/cuda-feedstock/issues/29

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,27 +9,27 @@ source:
   sha256: e2c71babfd18a8e69542dd7e9ca018f9caa438094001a58e6bc4d8c999bf0d07
 
 build:
-  number: 1
+  number: 0
   skip: true  # [osx]
```
Comment (jakirkham) [inline recipe/meta.yaml:13]:
```suggestion
  skip: true  # [osx or ppc64le]
```

**Review by jakirkham — COMMENTED** [review]
Thanks Alex! 🙏

Made a minor suggestion below

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,27 +9,27 @@ source:
...
-    - libnvjpeg-dev 12.3.1.117
+    - cuda-cudart-dev 12.5.39
+    - cuda-cccl 12.5.39
+    - cuda-profiler-api 12.5.39
+    - cuda-driver-dev 12.5.39    # [linux]
+    - cuda-nvrtc-dev 12.5.40
+    - cuda-opencl-dev 12.5.39    # [linux64 or win]
+    - libcublas-dev 12.5.2.13
+    - libcufft-dev 11.2.3.18
+    - libcufile-dev 1.10.0.4       # [linux64 or aarch64]
```
Comment (jakirkham) [inline recipe/meta.yaml:25]:
```suggestion
    - libcufile-dev 1.10.0.4       # [linux]
```

**Comment by adibbley** [comment]
@conda-forge-admin , please re-render

**Review by jakirkham — APPROVED** [review]

**Pull request merged**